### PR TITLE
start: faster to value and...

### PR DESCRIPTION
**Pull request by jorgeorpinel**
- and small updates to the home page;
- plus other misc. improvements

### CODE REVIEW

File: content/docs/index.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # CML Documentation
 
-[Continuous Machine Learning (CML)](https://cml.dev) is an open-source library
-for implementing continuous integration & delivery (CI/CD) in machine learning
+[Continuous Machine Learning (CML)](/) is an open-source library for
```
Comment (casperdcl) [inline content/docs/index.md:3]:
```suggestion
[Continuous Machine Learning (CML)](/) is an open-source tool for
```

File: content/docs/index.md [hunk 2]
```diff
@@ -35,3 +35,21 @@ us a ⭐ if you like the project!
...
+- **Auto reports for ML experiments.** Auto-generate reports with metrics and
+  plots in each Git Pull Request. Rigorous engineering practices help your team
+  make informed, data-driven decisions.
+
+- **No additional services.** Build your own ML platform using just GitHub or
+  GitLab and your [favorite cloud services]: AWS, Azure, GCP, or Kubernetes. No
+  databases, services or complex setup needed
+
+[gitflow]: https://nvie.com/posts/a-successful-git-branching-model
+[favorite cloud services]:
```
Comment (casperdcl) [inline content/docs/index.md:54]:
```suggestion
  GitLab and your [favorite cloud providers]: AWS, Azure, GCP, or Kubernetes. No
  databases, services or complex setup needed

[gitflow]: https://nvie.com/posts/a-successful-git-branching-model
[favorite cloud providers]:
```

File: content/docs/start/index.md [hunk 3]
```diff
@@ -1,36 +1,25 @@
...
-  GitLab and your
-  [favorite cloud services](/doc/self-hosted-runners#cloud-compute-resource-credentials):
-  AWS, Azure, GCP, or Kubernetes. No databases, services or complex setup
-  needed.
+# Get Started with CML
 
-_Need help? Just want to chat about continuous integration for ML?
-[Visit our Discord channel!](/chat)_
+On every pull request, CML helps you automatically train and evaluate models,
+then generates a visual report with results and metrics.
```
Comment (casperdcl) [inline content/docs/start/index.md:4]:
```suggestion
and make reports from results, plots and metrics.
```

File: content/docs/start/index.md [hunk 4]
```diff
@@ -1,36 +1,25 @@
...
-for hands-on MLOps tutorials using CML! 🌟
+https://www.youtube.com/watch?v=9BgIDqAzfuA&list=PL7WG7YrwYcnDBDuCkFbcyjnZQrdskFsBz&index=1&ab_channel=DVCorg
+
+[youtube video series]:
+  https://www.youtube.com/playlist?list=PL7WG7YrwYcnDBDuCkFbcyjnZQrdskFsBz
+
+</admon>
+
+_Need help? Just want to chat about continuous integration for ML?
+[Visit our Discord channel!](https://cml.dev/chat)_
```
Comment (casperdcl) [inline content/docs/start/index.md:23]: probably also admon?

**Review by casperdcl — CHANGES_REQUESTED** [review]

File: content/docs/start/index.md (hunk 4 above)
Comment (jorgeorpinel) [reply content/docs/start/index.md:23]:
```suggestion
<ad,on type="info">

_Need help? Just want to chat about continuous integration for ML?
[Visit our Discord channel!](https://cml.dev/chat)_

</admon>
```

File: content/docs/start/index.md [hunk 5]
```diff
@@ -1,36 +1,29 @@
...
+Check out our [YouTube video series] for hands-on MLOps tutorials using CML!
+
+https://www.youtube.com/watch?v=9BgIDqAzfuA&list=PL7WG7YrwYcnDBDuCkFbcyjnZQrdskFsBz&index=1&ab_channel=DVCorg
+
+[youtube video series]:
+  https://www.youtube.com/playlist?list=PL7WG7YrwYcnDBDuCkFbcyjnZQrdskFsBz
+
+</admon>
+
+<ad,on type="info">
```
Comment (jorgeorpinel) [inline content/docs/start/index.md:22]: Oops

```suggestion
<admon type="info">
```

File: content/docs/start/index.md [hunk 6]
```diff
@@ -1,36 +1,29 @@
...
-  make informed, data-driven decisions.
-- **No additional services.** Build your own ML platform using just GitHub or
-  GitLab and your
-  [favorite cloud services](/doc/self-hosted-runners#cloud-compute-resource-credentials):
-  AWS, Azure, GCP, or Kubernetes. No databases, services or complex setup
-  needed.
+# Get Started with CML
+
+On every pull request, CML helps you automatically train and evaluate models,
+and make reports from results, plots and metrics.
```
Comment (jorgeorpinel) [inline content/docs/start/index.md:4]:
```suggestion
On every pull request, CML helps you automatically train and evaluate machine
learning models, and make reports from results, plots and metrics.
```

File: content/docs/start/index.md [hunk 7]
```diff
@@ -1,36 +1,29 @@
...
+# Get Started with CML
+
+On every pull request, CML helps you automatically train and evaluate machine
+learning models, and make reports from results, plots and metrics.
+
+![](/img/cml_neural_transfer.png) _Report for a [neural style transfer model]._
+
+**Pick one of the supported platforms to continue: [GitHub](/doc/start/github) /
+[GitLab](/doc/start/github) / [Bitbucket](/doc/start/github)**
+
```
Comment (jorgeorpinel) [inline content/docs/start/index.md:10]: Should we move this near the top instead? (before the CML report image)

File: content/docs/index.md [hunk 8]
```diff
@@ -35,3 +35,21 @@ us a ⭐ if you like the project!
 
 ✅ Contribute to CML [on GitHub](https://github.com/iterative/cml) or help us
 improve this [documentation](https://github.com/iterative/cml.dev) 🙏.
+
+## CML principles
+
+- **[GitFlow] for data science.** Use GitLab or GitHub to manage ML experiments,
+  track who trained ML models, or modified data and when. Codify data and models
```
Comment (jorgeorpinel) [inline content/docs/index.md:42]: Moved these here (currently in https://cml.dev/doc/start).

File: content/docs/start/index.md [hunk 9]
```diff
@@ -1,36 +1,29 @@
...
+# Get Started with CML
+
+CML helps you automatically train and evaluate machine learning models right in
+your pull/merge requests. It can also embed reports from the results, including
+plots and metrics.
+
+![](/img/cml_neural_transfer.png) _Report for a [neural style transfer model]._
+
+**Pick one of the supported platforms to continue: [GitHub](/doc/start/github) /
+[GitLab](/doc/start/github) / [Bitbucket](/doc/start/github)**
```
Comment (jorgeorpinel) [inline content/docs/start/index.md:10]: Made the index very brief and clarified what CML does concretely along the way.

> Still don't love that you have to scroll a bit (after the image) to find links to each Git hosting platform but I'm not sure how to do that better.

**Review by jorgeorpinel — COMMENTED** [review]
Here's another contribution that may be good to consider. The goal was to make the Get Started more concise and for the reader to get to actionable items quickly, so that more people reach the "Aha!" moment. Cc @shcheklein 

Summary of changes (just 2 things):

**Review by 0x2b3bfa0 — APPROVED** [review]

File: content/docs/index.md [hunk 10]
```diff
@@ -35,3 +35,21 @@ us a ⭐ if you like the project!
...
+  with [DVC](/doc/cml-with-dvc) instead of pushing to a Git repo.
+
+- **Auto reports for ML experiments.** Auto-generate reports with metrics and
+  plots in each Git Pull Request. Rigorous engineering practices help your team
+  make informed, data-driven decisions.
+
+- **No additional services.** Build your own ML platform using just GitHub or
+  GitLab and your [favorite cloud providers]: AWS, Azure, GCP, or Kubernetes. No
```
Comment (shcheklein) [inline content/docs/index.md:50]: Can be BB also?

**Review by shcheklein — APPROVED** [review]
Thanks @jorgeorpinel . A minor comment, good to go besides that.

File: content/docs/index.md (hunk 10 above)
Comment (jorgeorpinel) [reply content/docs/index.md:50]: Yes, I think BitBucket is fully supported in every feature last time I checked.

**Pull request merged**